mlpipeline/src/mlpipeline/pretrain/model.py

```
@@ -83,16 +84,16 @@ class RecipeEncoder(nn.Module):
     def __init__(self, cfg: RecipeEncoderConfig):
         super().__init__()
         self.cfg = cfg
-        self.base_model = SentenceTransformer(cfg.st_model_name)
-        base_dim = self.base_model.get_sentence_embedding_dimension()
+        self.base_model = TextEmbedding(cfg.st_model_name)
+        base_dim = 384
 
         if base_dim != cfg.output_dim:
             print(f"Attention!!! the sentence transformer output dim is not equal to the out_dim")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)